File: Phonebook.web/ClientApp/src/app/models/phonebook-entry.ts
export class PhonebookEntry {
  Id: number;
  Name: string;
  PhoneNumber: string;

  constructor() {
    this.Id = 0;
    this.Name = '';
    this.PhoneNumber = '';

  }
}


export interface IPhonebookEntry {
  id: number;
  name: string;
  phoneNumber: string;
}

File: Phonebook.Contracts/Models/PhoneBookEntries/PhoneBookEntry.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Phonebook.Contracts.Models.PhoneBookEntries
{
    public class PhonebookEntry
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string PhoneNumber { get; set; }
        public int PhoneBookId { get; set; }
        public virtual Phonebooks PhoneBook { get; set; }


    }
}
